examples/error_handling/handle_partial_failure.py

```diff
@@ -53,58 +92,40 @@ def main(client, customer_id, campaign_id):
     ad_group_op3.create.campaign.value = resource_name
     ad_group_operations.append(ad_group_op3)
 
-    try:
-        # Issue a mutate request, setting partial_failure=True.
-        ad_group_response = ad_group_service.mutate_ad_groups(
-            customer_id, ad_group_operations, partial_failure=True)
-    except GoogleAdsException as ex:
-        print('Request with ID "%s" failed with status "%s" and includes the '
-              'following errors:' % (ex.request_id, ex.error.code().name))
-        for error in ex.failure.errors:
-            print('\tError with message "%s".' % error.message)
-            if error.location:
-                for field_path_element in error.location.field_path_elements:
-                    print('\t\tOn field: %s' % field_path_element.field_name)
-        sys.exit(1)
-    else:
-        # Check for existence of any partial failures in the response.
-        if ad_group_response.partial_failure_error:
-            print('Partial failures occurred. Details will be shown below.\n')
-            partial_failures = get_partial_failure_messages(
-                client, ad_group_response)
+    # Issue a mutate request, setting partial_failure=True.
+    return ad_group_service.mutate_ad_groups(customer_id, ad_group_operations,
+                                             partial_failure=True)
 
-            for error in partial_failures:
-                # Construct and print a string that details which element in
-                # the above ad_group_operations list failed (by index number)
-                # as well as the error message and error code.
-                print('A partial failure at index %d occured.\n'
-                      'Error message: %s\nError code: %s' % (
-                          error.location.field_path_elements[0].index.value,
-                          error.message, error.error_code))
-        else:
-            print('All operations completed successfully. No partial failure '
-                  'to show.')
+def check_if_partial_failure_exists(response):
```

Comment:
this function is awkwardly named. Maybe just `partial_failure_occurred` or just `partial_failure`.

This simplifies the later code `if partial_failure_occurred ...` instead of `if check_if_partial_failure_exists...`

Reply:
I do agree but I defer to @fiboknacky because the name of the method is consistent with the other examples. @fiboknacky is it OK if we rename this method to something like David describes or does it need to be named the way it is now?

Reply:
It's fine to change, but please make sure you modify accordingly in the DevSite guide.

Reply:
BTW, partial_failure_occurred or partial_failure_exists seems most natural to me.

Reply:
Done, went with `is_partial_failure_error_present` because it's nice for functions that return a boolean like this to start with `is`.